sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -279,19 +279,6 @@ public static bool IsStringEmpty(this ExpressionSyntax expression, SemanticModel
         public static Location FindIdentifierLocation(this BaseMethodDeclarationSyntax methodDeclaration) =>
             GetIdentifierOrDefault(methodDeclaration)?.GetLocation();
 
-        public static bool IsCatchingAllExceptions(this CatchClauseSyntax catchClause)
```

Comment:
this wasn't used?

Reply:
Indeed, dead code. 

Don't know yet what's the best approach when removing these. A separate PR or mix  with the PR in progress?

Reply:
ideally it would be a separate commit, to keep the history clean.